packages/debug/src/browser/debug-frontend-application-contribution.ts

```diff
@@ -797,8 +826,8 @@ export class DebugFrontendApplicationContribution extends AbstractViewContributi
                     await selectedFunctionBreakpoint.open();
                 }
             },
-            isEnabled: () => !!this.selectedBreakpoint || !!this.selectedFunctionBreakpoint,
-            isVisible: () => !!this.selectedBreakpoint || !!this.selectedFunctionBreakpoint
+            isEnabled: () => this.getSelectedBreakpoints().length === 1 && (!!this.selectedBreakpoint || !!this.selectedFunctionBreakpoint),
+            isVisible: () => this.getSelectedBreakpoints().length === 1 && (!!this.selectedBreakpoint || !!this.selectedFunctionBreakpoint)
```

Comment:
This is also true for the other commands: Do we really need those additional checks? It seems that the original checks should still be enough, what do you think?

Reply:
Well, the difference is whether the single-element-related menu items are shown in case of multi-selection.

Without the length check, the menu looks like this:
![Bildschirmfoto 2025-06-17 um 18 23 27](https://github.com/user-attachments/assets/b394c8d1-27d9-42ce-afe1-a05022620c77)

and with the length check:

![Bildschirmfoto 2025-06-17 um 18 29 52](https://github.com/user-attachments/assets/b5c2ea61-3d7b-4a39-83e4-59eda0121347)

My feeling is that showing single-element-related menu items for multi-selection (e.g. edit) is not intuitive, as it is not clear which breakpoint the user will actually be editing. So I have kept the length check for now.

Please let me know if you feel differently.

Reply:
Ah, you are right, my bad!